src/components/Table/Table.js

```diff
@@ -17,6 +17,7 @@ const Table = ({
       "p-table--sortable": sortable,
       "p-table-expanding": expanding
     })}
+    style={{ captionSide: "bottom" }}
```

Comment:
Cool style, I didn't know about this one!

Curious why this is here and not part of the `MainTable.scss`?